src/client/src/commons/form/profile/components/profileInstrument/components/instrument/Instrument.js

```diff
@@ -57,7 +59,7 @@ const Instrument = props => {
 
   const updateInstrument = (attribute, value) => {
     if (!isEditable) {
-      alert(t("common:errorStartEditing"));
+      alertContext.error(t("common:errorStartEditing"));
```

Comment:
Q: Wieso wird manchmal `this.context.error` und manchmal `alertContext.error` verwendet?

Reply:
Bei den alten React Class Components muss der Context anders importiert werden. Der useContext hook wie bei den neuen Functional Components geht da nicht.